pkg/controller/node/BUILD

```diff
@@ -38,6 +38,7 @@ go_library(
         "//pkg/cloudprovider:go_default_library",
         "//pkg/cloudprovider/providers/gce:go_default_library",
         "//pkg/controller:go_default_library",
+        "//pkg/kubelet:go_default_library",
```

Comment:
Creating false dependencies between packages just for constants is bad.  If you only need constants, create a new constants package. However, I'd push back on why retry constants are shared between master and node - they are always asymmetric, and I don't know what's special about this constant that means it has to always be the same